oelint_adv/cls_rule.py

```diff
@@ -97,17 +96,18 @@ def finding(self, _file, _line, override_msg=None, appendix=None):
         if _line <= 0:
             # Fix those issues, that don't come with a line
             _line = 1
+
```

Comment:
Not needed

Reply:
Maybe, but better groups blocks of logic. I would actually prefer to see many more blank lines within the code as it eases the perception of what belongs togehter.

Reply:
Well well well, then I'd like to see a automated way of checking that, do you happen to know one?
Again here I have a slightly different view on that topic :wink:, usually I like my code to be as compact as possible, as one can see then a larger context on a single screen.
If you can come up with a checker how to enforce such a metric in CI, I'd be fine with adding more new lines.

On an unrelated topic: you should cleanup you commit messages here

Reply:
I also found the use of blank lines a very natural choice to separate logic blocks and felt like most other projects use the same. Also, to be honest, I find the point that a method is more likely to fit into a terminal window complete rather weak (-> What's your screen size? whats your terminal font size?)
The usage of blank lines is also recommended in PEP8 (though 'sparingly' leaves a lot of room for interpretation)
https://www.python.org/dev/peps/pep-0008/#blank-lines

And there is tons of more articles that recommend blank lines to separate blocks of logic. I will remove them if you insist but it'll make the code harder to read (and that's what matters to me the most).

--

I'm usually squashing commits before final reviews, especially when taming the CI as this often creates a bit of noise.